saleor/extensions/base_plugin.py

```diff
@@ -113,47 +171,97 @@ def apply_taxes_to_product(
         country: Country,
         previous_value: TaxedMoney,
     ) -> TaxedMoney:
+        """Apply taxes to the product price based on the customer country.
+
+        Overwrite this method if you want to show products with taxes.
+        """
         return NotImplemented
 
     def preprocess_order_creation(
         self, checkout: "Checkout", discounts: List["DiscountInfo"], previous_value: Any
     ):
+        """Trigger directly before order creation.
+
+        Overwrite this method in case you need to trigger specific logic before the
+        order will be created.
+        """
         return NotImplemented
 
     def order_created(self, order: "Order", previous_value: Any):
+        """Trigger when order is created.
+
+        Overwrite this method in case you need to trigger specific logic after order
+        creation.
+        """
         return NotImplemented
 
     def assign_tax_code_to_object_meta(
         self, obj: Union["Product", "ProductType"], tax_code: str, previous_value: Any
     ):
+        """Assign tax code dedicated to plugin."""
         return NotImplemented
 
     def get_tax_code_from_object_meta(
         self, obj: Union["Product", "ProductType"], previous_value: "TaxType"
     ) -> "TaxType":
+        """Return tax code from object meta."""
         return NotImplemented
 
     def get_tax_rate_percentage_value(
         self, obj: Union["Product", "ProductType"], country: Country, previous_value
     ) -> Decimal:
+        """Return tax rate percentage value for given tax rate type in the country.
+
+        It is used only on the old storefront.
+        """
         return NotImplemented
 
     def customer_created(self, customer: "User", previous_value: Any) -> Any:
+        """Trigger when user is created.
+
+        Overwrite this method in case you need to trigger specific logic after user
+        creation.
+        """
         return NotImplemented
 
     def product_created(self, product: "Product", previous_value: Any) -> Any:
+        """Trigger when product is created.
+
+        Overwrite this method in case you need to trigger specific logic after product
+        creation.
+        """
         return NotImplemented
 
     def order_fully_paid(self, order: "Order", previous_value: Any) -> Any:
+        """Trigger when order is fully-paid.
+
+        Overwrite this method in case you need to trigger specific logic when the order
+        is fully-paid.
+        """
         return NotImplemented
 
     def order_updated(self, order: "Order", previous_value: Any) -> Any:
+        """Trigger when order is updated.
+
+        Overwrite this method in case you need to trigger specific logic when order is
+        changed.
+        """
         return NotImplemented
 
     def order_cancelled(self, order: "Order", previous_value: Any) -> Any:
+        """Trigger when order is cancelled.
+
+        Overwrite this method in case you need to trigger specific logic when order is
```

Comment:
Overwrite this method if you need to trigger specific logic when an order is canceled.